CMakeLists.txt

```diff
@@ -9,20 +9,13 @@ set (CXX_WARNING_FLAGS "-Wnon-virtual-dtor")
 
 set (CMAKE_BUILD_COLOR_MAKEFILE          ON)
 set (CMAKE_CXX_FLAGS                     "${CMAKE_CXX_FLAGS} -std=c++14 -O2 -g -fvisibility-inlines-hidden")
-set (CMAKE_SHARED_LINKER_FLAGS           "${CMAKE_SHARED_LINKER_FLAGS} -Wl,-Bstatic,--whole-archive -static-libgcc -static-libstdc++ -Wl,--no-whole-archive")
```

Comment:
It brakes driver's work in Ubuntu Linux. Can't accept the patch.

Reply:
travis-ci (which is ubuntu) does not display any issue
how to reproduce the issue ?

Reply:
>>> travis-ci (which is ubuntu) does not display any issue
Because there is no tests for .so itself.

>>> how to reproduce the issue ?
Just build the driver and try to use it.